package.json

```diff
@@ -31,6 +31,7 @@
   "dependencies": {
     "@types/compression": "^0.0.33",
     "@types/express": "^4.0.35",
+    "@types/http-errors": "^1.6.1",
```

Comment:
Also add `http-errors` itself to dependencies (not just the typings). It works anyway because there's a transitive dependency via `send`, but it's better to declare it directly too now that we use it.

Reply:
Done.